pkg/log/null.go

```diff
@@ -25,23 +29,46 @@ import (
 // all binaries.
 
 // NullLogger is a logr.Logger that does nothing.
-type NullLogger struct{}
+type NullLogger struct {
+	warnOnce
+}
+
+// warnOnce warn once when NullLogger is called
+type warnOnce struct {
+	once      *sync.Once
+	warnPrint string
+}
+
+func (w warnOnce) do() {
+	w.once.Do(func() {
+		fmt.Fprintln(os.Stdin, w.warnPrint)
+	})
+}
 
 var _ logr.Logger = NullLogger{}
 
+// DefaultNullLogger a default NullLogger with specified warn print
+func DefaultNullLogger() NullLogger {
```

Comment:
Why export this?

Reply:
Sorry for disconnection for a long while! 
I abandoned this intrusive way, used a simple way to print error log.